diff --git a/Kconfig.sources b/Kconfig.sources
index b68b2c3..1e59c1a 100644
--- a/Kconfig.sources
+++ b/Kconfig.sources
@@ -4,14 +4,14 @@ source "$BACKPORT_DIR/compat/Kconfig"
 # these are copied from the kernel
 source "$BACKPORT_DIR/drivers/dma-buf/Kconfig"
 
-source "$BACKPORT_DIR/drivers/gpu/drm/Kconfig"
+#source "$BACKPORT_DIR/drivers/gpu/drm/Kconfig"
 
-source "$BACKPORT_DIR/drivers/gpu/drm/i915/Kconfig"
+#source "$BACKPORT_DIR/drivers/gpu/drm/i915/Kconfig"
 
 # PMT
-source "$BACKPORT_DIR/drivers/mfd/Kconfig"
-source "$BACKPORT_DIR/drivers/platform/x86/intel/Kconfig"
+#source "$BACKPORT_DIR/drivers/mfd/Kconfig"
+#source "$BACKPORT_DIR/drivers/platform/x86/intel/Kconfig"
 
 # MEI
-source "$BACKPORT_DIR/drivers/misc/mei/Kconfig"
-source "$BACKPORT_DIR/drivers/watchdog/Kconfig"
+#source "$BACKPORT_DIR/drivers/misc/mei/Kconfig"
+#source "$BACKPORT_DIR/drivers/watchdog/Kconfig"
